README.md

```diff
@@ -69,6 +69,29 @@ are generated from a central template. To edit one of these files, make an edit
 to its template in this
 [directory](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library).
 
+## Deployment
+
+The building and deployment process of Flaky.dev has been automated using Cloud Build triggers.  The build steps are outlined in the following two files: `cloudbuild-api.yaml` located in `packages/api` and `couldbuild-frontend.yaml` located in `packages/frontend`.  We have implemented a two-pipeline deployment process:
```

Comment:
I might be tempted to link to the file here, you should be able to just do:

```md
files: [`packages/api/cloudbuild-api.yaml`](./packages/api/cloudbuild-api.yaml
```

Reply:
Good Idea! Will do